src/constants/constants.js

```diff
@@ -0,0 +1,13 @@
+/************************************* COLORS ***********************************************/
+export const black = "#333333";
+export const white = "#FFFFFF";
+
+export const primary900 = "#361C4E";
```

Comment:
The name of the color is not very clear to use. Better to name like primary-black or primary-orange.

Reply:
since would we make a difference between the shades since it goes from 900 to 100 ?